src/authorizeStep.js

```diff
@@ -18,8 +18,8 @@ export class AuthorizeStep {
         return next.cancel(new Redirect(loginRoute));
       }
     }
-    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment) == loginRoute) {
-      var loginRedirect = this.auth.getLoginRedirect();
+    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment == loginRoute)) {
+      var loginRedirect = this.auth.getLoginRedirect().replace(/\/#/gi, '');
```

Comment:
Sorry this wasn't suppose to be in this pull request. 

But the replace needs to be there anyways since the login redirect route is in the format of "/#customers" or "/#/customers" then new Redirect() doesn't see it as a route and it doesn't work. It needs to be in the format of "customers" or "/customers" it only works on login since the redirect is using window.href  =